src/lib.rs

```diff
@@ -31,6 +31,23 @@ fn main() {
     assert_eq!(*foo.private(), 2);
 }
 ```
+```compile_fail
```

Comment:
This needs some related documentation or something. :)

Reply:
Added documentation about struct level atrributes here

Reply:
I think the `compile_fail` could be reduced to this:

```rust
    #[derive(Getters, Default)]
    #[get = "pub"]
    pub struct Foo {
        #[get]
        private: i32,
        public: i32,
    }
```

Then maybe we can have a full, valid example of the struct level attributes and field level working?

Reply:
It's compile_fail test not because we cannot mix struct and field level attributes. It's fine.
But generated getter is private, and we try to call it and this is compile time error.
I clarify this is with comment
>Sorry to be such a stickler about this
It's fine